app-text/scdoc/Manifest

```diff
@@ -1 +1 @@
-DIST scdoc-1.3.1.tar.xz 8776 BLAKE2B 1d11423abde7ec3e7b03ddc4f9698b7b78cac6972066c57886160974bd885225dc42793d19942c11ffd215f5dddd6453f66cbd083be03132b437b6f6f8a93c8a SHA512 a404df2e2eef76cc02c69f32eb131eaa4d4ed51e96dc6ea39400662f767648538fc27c618cf34c064004918a92940ae5400e91b7b628e431132cb9619d3e94eb
+DIST scdoc-1.3.1.tar.gz 9392 BLAKE2B 609f25252f0ee864ba3500961aa1048c6490278f4861fb4494ca62365a20d0cb128e53a6397bd089bd5a37f25f7f3c080f6921f1c4f4ca7e4af729c87e1994fc SHA512 6064de8c76eb959708b32e93634223565e5d4b390069d4fcd2d87d27bb472a333c60b21de87d676eb57ff5f8ed6445d3c3bd92dfe7dd611f9886abc38e8d21d2
```

Comment:
Have you verified that the contents are identical?

Reply:
Untarred both, got a listing of files, and used fdupes against both directories to check
hashes. To be fair I don't know what hash fdupes uses, so there is a possibility of a
collision if it uses a weak one but that's hella rare.

Reply:
It would be simpler to use `diff -R` ;-P.